AppCenter.podspec

```diff
@@ -1,4 +1,5 @@
 Pod::Spec.new do |s|
+  s.cocoapods_version = '>= 1.10'
```

Comment:
do we need to update the changelog that we raised the minimal version of cocoapods?
Also please update the template in PR description.

Reply:
done!

Reply:
did you test your changes? why it doesn't mention in the template?

Reply:
yes, now it's correct!